openapi.yaml

```diff
@@ -463,6 +464,77 @@ paths:
+
+        ```JSON
+        POST https://api.<region>.whispir.com/workspaces/{workspaceId}/messages
+         
+        Authorization: Basic am9obi5zbWl0aDpteXBhc3N3b3Jk
+        x-api-key: your_api_key
+        Content-Type: application/vnd.whispir.message-v1+json
+        Accept: application/vnd.whispir.message-v1+json
+        In this sample we will send a message with an opt out option property
+        {
+            "to" : "jsmith@test.com",
+            "subject" : "Test Opt-out and Unsubscribe",
+            "email" : {
+                "body" : "This is a test message",
+                "type" : "text/plain"
+                "optOutOption": {
+                  "rmsUnsubscribeUrl": "@@web_link@@#unsubscribe",
+                  "oneClickApiMapping": "2ab64862"
+                }
+            },
+            "web": {
+              "type": "text/html",
+              "body": "<div class='content'>This is the body of my rich message</div>"
+            }
+        }
+        ```
+        ```XML
+        POST https://api.<region>.whispir.com/workspaces/{workspaceId}/messages
+
+        Authorization: Basic am9obi5zbWl0aDpteXBhc3N3b3Jk
+        x-api-key: your_api_key
+        Content-Type: application/vnd.whispir.message-v1+xml
+        Accept: application/vnd.whispir.message-v1+xml
+        In this sample we will send a message with an opt out option property
+        <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+        <ns2:message xmlns:ns2="http://schemas.api.whispir.com" xmlns:ns3="http://schemas.api.whispir.com/dap">
+          <to>john.smith@test.com</to>
+          <subject>Test scheduled email message</subject>
+          <email>
+              <body>This is my scheduled content</body>
+              <type>text/plain</type>
+              <optOutOption>
+                <rmsUnsubscribeUrl>@@web_link@@#unsubscribe</rmsUnsubscribeUrl>
+                <oneClickApiMapping>unsubscribe</oneClickApiMapping>
+              </optOutOption>
+          </email>
+          <web>
+            <type>text/html</type>
+            <body>
+              <div class='content'>This is the body of my rich message</div>
+            </body>
+          </web>
+        </ns2:message>
+        ```
+
+        In this example we're utilizing the `@@web_link@@` tag that generates an unsubscribe link for the recipients.
+        The `web` payload is required when using the `@@web_link@@` value in the `email`.`optOutOption`.
+        
+        The `rmsUnsubcribeUrl` is a URL link to the web channel unsubscribe content for `GET` requests, and for one-click unsubscribe functionality via `POST` requests.
+
+        The `oneClickApiMapping` property requires you to have an existing API Mapping and have access to its details. This API Mapping is triggered for one-click unsubscribe functionality.
+        
+        Both the `rmsUnsubscribeUrl` and `oneClickApiMapping` properties are optional.
```

Comment:
both are optional, but rmsUnsubscribeUrl is required to enable unsubscribe and oneClickApiMapping is required to enable one-click unsubscribe

Reply:
I'll resolve this comment :) @lnachor-whispir